cardigan/stories/components/Cards/GuideStopCard/GuideStopCard.stories.tsx

```diff
@@ -34,6 +34,14 @@ const meta: Meta<typeof GuideStopCard> = {
       },
     },
   },
+  parameters: {
+    gridSizes: {
+      s: [12],
+      m: [6],
+      l: [4],
+      xl: [4],
+    },
+  },
```

Comment:
Not for this PR but should we be using set grid sizes in Cardigan?

Reply:
I guess we just use the ones that are most appropriate for a given component at a given viewport width? So not sure we can/should decide what they should be at a higher level (although I might have not understood what you mean)